src/components/SearchBar/SearchBar.js

```diff
@@ -1,8 +1,20 @@
+import React, { useState } from 'react'
 import './SearchBar.css';
 
 const SearchBar = () => {
+
+  const [ searchInput, setSearchInput ] = useState('');
```

Comment:
This is the right starting point for this state. Next, you'll need to lift it up. This is because we need to use the search input to filter our list of users.

To filter the list of users, either the `<Users />` component or a parent of `<Users />` (`<App />`) will need to know what the user typed. The state variable `searchInput` currently holds what the user typed. That state is currently inside `<SearchBar />`. However, `<SearchBar />` is not the same component as `<Users />` and is not a parent of `<Users />`. So we need to make the info in `searchInput` accessible to `<Users />` or a parent of `<Users />`. That means lifting state up to a common parent component, in this case `<App />`.